scripts/add-pr-comment.mjs

```diff
@@ -193,4 +193,5 @@ const resultText = getResultText(updatedWorkspaces)
 const response = await addPrComment(resultText)
 console.log(response)
 
+console.log()
```

Comment:
is this intentional?

Reply:
Yeah, just an extra newline I wanted in there. I could switch it to `console.log(response + '\n')` if you think that'd make more sense.